script/automerge.ts

```diff
@@ -0,0 +1,99 @@
+require('dotenv-safe').load()
+
+import * as Octokit from '@octokit/rest';
+
+const OWNER = 'electron'
+const REPO = 'i18n';
+const BOTNAME = 'glotbot'
+const SEMANTIC_TITLE = 'feat: New Crowdin translations (auto-merging 🤖)';
+
+const github = new Octokit({
+  auth: process.env.GITHUB_TOKEN
+})
+
+const getPRData = async (prNumber: number) => {
+  return github.pulls.get({
+    owner: OWNER,
+    repo: REPO,
+    pull_number: prNumber
+  })
+}
+
+/**
+ * Gets the number of opened pull request by the bot or user.
+ */
+const getPRNumber = async () => {
+  const prs = await github.pulls.list({
+    owner: OWNER,
+    repo: REPO
+  })
+  const glotbot = await prs.data.filter(pr => pr.user.login === BOTNAME)
+  const prNumber = glotbot[0].number
+  return prNumber
+}
+
+/**
+ * Updates the default Crowding title with sematic title for
+ * pass all checks before merge.
+ *
+ * @param pr The number of pull request. Takes automatically
+ *           in autoMerger() function.
+ */
+const updateTitle = async (pr: number) => {
+  const semanticTitle = await github.pulls.update({
+    owner: OWNER,
+    repo: REPO,
+    pull_number: pr,
+    title: SEMANTIC_TITLE
+  })
+  return semanticTitle
+}
+
+/**
+ * Checks if the pull request able to merge.
+ *
+ * @param pr The number of pull request. Takes automatically
+ *           in autoMerger() function.
+ */
+const ableToMerge = async (pr: number) => {
```

Comment:
I think this would be clearer if it returned `true` or `false` and we checked the value inside `autoMerger`